app/(frontend)/auth/login/page.tsx

```diff
@@ -3,15 +3,36 @@
 import Image from "next/image"
 import {Button, Input} from "@nextui-org/react";
 import {Query} from "@/app/lib/api/Query";
+import {useRouter} from "next/navigation";
+import {useState} from "react";
+import {ILoginFormError} from "@/app/lib/api/error/ApiError";
+import Link from "next/link";
+import {useCookies} from "react-cookie";
+import {AUTHORIZATION} from "@/app/lib/definitions";
+
+const DAYS = 1
+const MAXAGE = 60 * 60 * 24 * DAYS
```

Comment:
When using a variable's name is in full uppercase, then the naming convention should use snake case. Please refactor it, because MAX_AGE is much easier to read than MAXAGE.